ert_data/results.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
+from ert_data import loader
+from ert_data.measured import MeasuredData, _add_index_range
+
+
+class Results:
+
+    def __init__(self, facade, keys, case):
+        self._facade = facade
+        self._case = case
+        self._set_data(self._get_data(keys))
+
+    @property
+    def data(self):
+        return self._data
+
+    def _set_data(self, data):
```

Comment:
Don't understand how `_set_data` and `_get_data` refer to `self._data`

Reply:
They are called in the constructor. I think it should be changed to simpler code.